api-metastore/src/main/java/org/zalando/nakadi/controller/SchemaController.java

```diff
@@ -31,7 +40,21 @@ public SchemaController(final SchemaService schemaService, final EventTypeServic
         this.eventTypeService = eventTypeService;
     }
 
-    @RequestMapping("/event-types/{name}/schemas")
+    @RequestMapping(value = "/event-types/{name}/schemas", method = RequestMethod.POST)
+    public ResponseEntity<?> create(@PathVariable("name") final String name,
+                                    @Valid @RequestBody final EventTypeSchemaBase schema,
+                                    final Errors errors) {
+        if (errors.hasErrors()) {
+            throw new ValidationException(errors);
+        }
+
+        final EventType eventType = eventTypeService.get(name);
+        schemaService.addSchema(eventType, schema);
```

Comment:
It makes sense to pass event type name to `addSchema` call, instead of event type, that is takes from cache, not from database (or find another solution)